includes/class-sensei-frontend.php

```diff
@@ -1264,7 +1264,7 @@ public function sensei_course_start() {
 				if ( false !== $redirect_url ) {
 					?>
 
-					<script type="text/javascript"> window.location = '<?php echo esc_url( $redirect_url ); ?>'; </script>
+					<script type="text/javascript"> window.location = '<?php echo esc_url_raw( $redirect_url ); ?>'; </script>
```

Comment:
It was redirecting to a broken URL in a published course with only draft lessons.